config/msb/conf.d/imu.yaml

```diff
@@ -0,0 +1,10 @@
+verbose         : false     # print debugging information
+print_stdout    : false     # print data to stdout
+sample_rate_div : 31        # output data rate (odr) is calculated like this: odr = 1125 / (1 + sample_rate_div)
+acc_filter      : 0         # available filters go from 0 - 7
```

Comment:
This is not up to date with the IMUConf, acc_filter and gyr_filter should be strings mapping to enums.

Reply:
fixed it.